app/controllers/course/courses_controller.rb

```diff
@@ -13,6 +13,8 @@ def show # :nodoc:
     @registration = Course::Registration.new
     @currently_active_announcements = current_course.announcements.currently_active
     @activity_feeds = recent_activity_feeds.limit(20).includes(activity: [:object, :actor])
+
+    current_course_user&.update_attribute(:last_active_at, Time.zone.now)
```

Comment:
In that case almost all of our actions (show/create/update/...) will involve this, which I think it's costly and unnecessary. 

I think current solution is good enough. But If we want more accurate results, the other way is to adopt this to the index action of each components.  Thoughts ?